test/test_datasets.py

```diff
@@ -1737,5 +1723,20 @@ def inject_fake_data(self, tmpdir, config):
         return split_to_num_examples[config["train"]]
 
 
+class SvhnTestCase(datasets_utils.ImageDatasetTestCase):
+    DATASET_CLASS = datasets.SVHN
+    ADDITIONAL_CONFIGS = datasets_utils.combinations_grid(split=("train", "test", "extra"))
```

Comment:
I think we'll need to set the REQUIRED_PACKAGES attribute for the scipy dependency?